bin/git-ignore

```diff
@@ -14,7 +14,15 @@ function show_global {
 }
 
 function add_global {
-  add_patterns `git config --global core.excludesfile` "$@"
+  local global_gitignore=$(git config --global core.excludesfile)
+  if [ "x${global_gitignore}" == "x" ]; then
```

Comment:
we can use a modern style of empty string check:
`-z "$global_gitignore"` or `"$global_gitignore" = ""`